docs/UsersGuide.asciidoc

```diff
@@ -957,20 +957,26 @@ openQA frequently checks whether assets need to be removed according to
 the configured limits.
 
 To find out whether an asset should be removed, openQA determines by which
-job groups the asset is used. If at least one job within a certain job group
-is using an asset, the asset is considered to be used by that group.
+groups the asset is used. If at least one job within a certain job group is
+using an asset, the asset is considered to be used by that job group. If
+that job group is within a parent group, the asset is considered part of
+that parent group.
 
-So an asset can be accounted to multiple groups. The assets table which is
-accessible via the admin menu shows these groups for each asset and also
-the latest job.
+So an asset can belong to multiple job or parent groups. The assets
+table which is accessible via the admin menu shows these groups for each
+asset and also the latest job.
 
-If the size limit for assets by a certain group is exceeded, openQA will
-remove assets accounted to that group:
+While an asset might belong to multiple groups it is only *accounted*
+to the largest group which has still enough room to hold that asset. So
```

Comment:
"largest group" can be ambiguous, e.g. a job group with most child job groups or a job group with "most tests". Better be explicit about the size quota. And I am not sure about the meaning here. Is it that an asset is assigned to the job group that has the highest amount of free quota or the job group with the highest total quota value?

Reply:
This is explaining the algorithm and that can only go wrong - I just wouldn't describe that detail at all.

Reply:
> "largest group" can be ambiguous, e.g. a job group with most child job groups or a job group with "most tests".

None of those options. "largest group" means the group with the highest size limit for assets.

> Is it that an asset is assigned to the job group that has the highest amount of free quota or the job group with the highest total quota value?

As I said, "largest group" refers to "highest total quote" but there's of course the restriction "which has still enough room to hold that asset" so the "free quota" is obviously not ignored, too. Likely @coolo is right an we shouldn't describe the algorithm in that detail.